.releaserc.json

```diff
@@ -21,9 +21,19 @@
       "channel": "next"
     },
     {
-      "name": "demo",
+      "name": "feature/**",
```

Comment:
Does the semantic release accept wide matching? Do you have any document or example for this? I don't know if it does.